src/Compilers/CSharp/Portable/Lowering/SpillSequenceSpiller.cs

```diff
@@ -274,7 +270,7 @@ private BoundStatement UpdateStatement(BoundSpillSequenceBuilder builder, BoundS
                 builder.AddStatement(statement);
             }
 
-            var result = _F.Block(builder.GetLocals(), builder.GetStatements());
```

Comment:
It feels like we need to check other usages of this API and its overloads that pull syntax from the factory. Perhaps, we should add explicit syntax parameter to all of them, to make sure an explicit decision is made about syntax at each call site.

Reply:
I agree, although I'd like to do that separately from this PR in case this specific fix needs to be merged sooner. Logged [https://github.com/dotnet/roslyn/issues/53019](https://github.com/dotnet/roslyn/issues/53019).